content/blogs/newsletter-101.md

```diff
@@ -0,0 +1,69 @@
+---
+title:       "Newsletter 101 - July 2021 news"
+author:      "Extravert34"
+date:        2021-07-26
+tags:        [ newsletter ]
+---
+
+Hello ReactOS followers! This report covers changes in the project during February-July 2021.
+And we definitely have some things to highlight!
+
+## amd64 build is getting more stable
+Timo Kreuzer (tkreuzer) worked hard on various parts of the kernel and HAL, fixing issues here and there.
+Structured Exception Handling (SEH) support for the amd64 architecture was finished, various bugs around the kernel are fixed.
+A major issue with interrupt handling in HAL was also fixed in May, which finally allowed a semi-stable boot in a virtual environment.
+
+## i686 APIC HAL now works
+ReactOS historically uses a legacy HAL on i686 architecture, which works with good old Intel 8259 PIC's interfaces for interrupt handling.
+This works in a single CPU environment, but for multiprocessor support, a more modern APIC interrupt handling is required.
+Timo Kreuzer also did a great job here reviving old code for APIC support.
+APIC HAL can be installed now by choosing the corresponding menu entry in the installer.
+
+## SMP GSoC project
+While preparing for the project, Jérôme Gardou (zefklop) and Thomas Faber (thfabba) were testing the kernel build in SMP mode,
+but running on a VM with only one CPU assigned.
+Default single-CPU build of ReactOS removes some code which is not neccessary for an
+execution on such machines.
+Such SMP build revealed issues in kernel locking primitives and bugs in some drivers, which
+were successfully [fixed](https://github.com/reactos/reactos/pull/3672).
+
+// Justin, put some info about your project here
+
```

Comment:
```suggestion

## `iernonce` GSoC project
He Yang started to work on `iernonce` under the guidance of his mentor George Bișoc and Hermès Bélusca. ReactOS already supported `RunOnce` registry allowing command execution on machine startup/user login. `iernonce` will enable users to use `RunOnceEx` registry to execute multiple commands in order, with a dialog displaying the progress on machine startup/user login.
```

Reply:
I wonder, whether a screenshot could be added?

Reply:
Send me one ;)